### PR TITLE
Add caniuse mappings

### LINKED ISSUE #2519: possible missing caniuse mappings
Hi. there. Thank you for your very nice and helpful project.  
I'm not javascript developer, I just noticed that a few modernizr's tests haven't mappings to caniuse db. 

Left side    ==>> caniuse  
Right side ==>> modernizr  

**Possible** mappings:
```python
CANIUSE_TO_MODERNIZR_PATCH = {
    "audio": "audio",
    "ambient-light": "ambientlight",
    "css-background-offsets": "bgpositionshorthand",
    "rem": ["csschunit", "cssexunit"],
    "css-snappoints": "scrollsnappoints",
    "css-text-align-last": "textalignlast",
    "will-change": "willchange",
    "custom-elementsv1": "customelements",
    "passive-event-listener": "passiveeventlisteners",
    "template": "template",
    "gamepad": "gamepads",
    "iframe-sandbox": "sandbox",
    "iframe-srcdoc": "srcdoc",
    "apng": "apng",
    "jpeg2000": "jpeg2000",
    "webp": "webp",
    "input-search": "inputsearchevent",
    "internationalization": "intl",
    "pointer-events": "pointerevents",
    "speech-recognition": "speechrecognition",
    "speech-synthesis": "speechsynthesis",
    "urlsearchparams": "urlsearchparams",
    "vibration": "vibrate",
     "xhr2": "xhr2",
    "rtcpeerconnection": "peerconnection",
    "beacon": "beacon",
    "addeventlistener": "eventlistener",
    "serviceworkers": "serviceworker",
}
```

### PR DESCRIPTION
**Pull request by MarkelFe**
Add the mappings that @PavMel pointed out in #2519, so this would then close #2519

I only have doubts with [csschunit](https://github.com/Modernizr/Modernizr/pull/2535/files#diff-72a64464df4b2f8f4c8f5eb394d5a740) and [cssexunit](https://github.com/Modernizr/Modernizr/pull/2535/files#diff-94016d36369d08fc37d795f62ff398a6) in which the rem page of caniuse is used, but as far as I know they are not the same.

Co-Authored-By: PavMel <pavmel@users.noreply.github.com>

### CODE REVIEW

File: feature-detects/inputsearchevent.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {
   "name": "input[search] search event",
   "property": "inputsearchevent",
+  "caniuse": "input-search",
```
Comment (rejas) [inline feature-detects/inputsearchevent.js:5]: this looks like the element in the caniuse database while  modernizr tests if there a spoecial search event.. better remove this

File: feature-detects/pointerevents.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 {
   "name": "DOM Pointer Events API",
   "property": "pointerevents",
+  "caniuse": "pointer-events",
```
Comment (rejas) [inline feature-detects/pointerevents.js:5]: no, this caniuse tag refers to the CSS pointer-events not the DOM pointer-events (two different things and tests)

File: feature-detects/css/backgroundposition-shorthand.js [hunk 3]
```diff
@@ -2,6 +2,7 @@
 {
   "name": "Background Position Shorthand",
   "property": "bgpositionshorthand",
+  "caniuse": "css-background-offsets",
```
[the remark this answers is not in the document]
Comment (MarkelFe) [reply feature-detects/css/backgroundposition-shorthand.js:5]: Just checked the MDN Docs and they are the same property. But yes, they sound very different.

File: feature-detects/css/exunit.js [hunk 4]
```diff
@@ -3,6 +3,7 @@
   "name": "CSS Font ex Units",
   "authors": ["Ron Waldon (@jokeyrhyme)"],
   "property": "cssexunit",
+  "caniuse": "rem",
```
[the remark this answers is not in the document]
Comment (MarkelFe) [reply feature-detects/css/exunit.js:6]: Haven't been able to introduce it in the url without the #feat=, I don't know if that would result in a problem, adding it anyway

File: feature-detects/pointerevents.js (hunk 2 above)
Comment (MarkelFe) [reply feature-detects/pointerevents.js:5]: Okay I think that the one we are looking for is [pointer](https://caniuse.com/#feat=pointer)

**Comment by MarkelFe** [comment]
Okay, just made the changes to the properties you told me :)

**Comment by rejas** [comment]
Looks good, thx for the quick rework!

**Comment by MarkelFe** [comment]
> 
> 
> Looks good, thx for the quick rework!

Things of the quarantine 😅

**Pull request merged**